taskforce/apps/users/src/app/auth/rdo/logged-user.rdo.ts

```diff
@@ -4,7 +4,7 @@ import { ApiProperty } from '@nestjs/swagger';
 export class LoggedUserRdo {
   @ApiProperty({
     description: 'The uniq user ID',
-    example: '42d448f8-9111-4ad7-ac70-2b6dd34af25'
+    example: '638348b04f5f6091439ea5b2'
   })
   @Expose({name: '_id'})
   public id: string;
```

Comment:
что бы проблему с айдишниками решить нужно ещё один декоратор использовать, что то типа такого
```suggestion
  @Transform((value) => value.obj._id.toString())
  @Expose({name: '_id'})
  public id: string;
```